### ISSUE TITLE
Wrong implementation of focal loss

### ISSUE THREAD

**Issue by UmarSpa** [makalo/CornerNet#21]
> pos_loss=tf.reduce_sum(tf.log(pre) * tf.pow(pos_weight,2))
> neg_loss=tf.reduce_sum(tf.pow((1-gt),4)*tf.pow(neg_weight,2)*tf.log((1-pre)))

tf.log(pre) in pos_loss and tf.log((1-pre) in neg_loss are wrong.
